packages/palette/src/elements/Box/Box.tsx

```diff
@@ -5,81 +5,45 @@ import { styled as primitives } from "../../platform/primitives"
 import {
   background,
   BackgroundProps,
-  bottom,
-  BottomProps,
-  color as styledColor,
+  border,
+  BorderProps,
+  color,
   ColorProps,
   compose,
-  display,
-  DisplayProps,
-  height,
-  HeightProps,
-  left,
-  LeftProps,
-  maxHeight,
-  MaxHeightProps,
-  maxWidth,
-  MaxWidthProps,
-  minHeight,
-  MinHeightProps,
-  minWidth,
-  MinWidthProps,
+  flexbox,
+  FlexboxProps,
+  layout,
+  LayoutProps,
   position,
   PositionProps,
-  right,
-  RightProps,
-  space as styledSpace,
+  space,
   SpaceProps,
   textAlign,
   TextAlignProps,
-  top,
-  TopProps,
-  width,
-  WidthProps,
-  zIndex,
-  ZIndexProps,
 } from "styled-system"
 
 export interface BoxProps
   extends BackgroundProps,
-    BottomProps,
-    ColorProps,
-    DisplayProps,
-    HeightProps,
-    LeftProps,
-    MaxWidthProps,
-    MinWidthProps,
-    MaxHeightProps,
-    MinHeightProps,
+    BorderProps,
+    Omit<ColorProps, "color">,
+    FlexboxProps,
+    LayoutProps,
     PositionProps,
-    RightProps,
     SpaceProps,
-    TextAlignProps,
-    TopProps,
-    WidthProps,
-    ZIndexProps {}
+    TextAlignProps {}
 
 /**
  * All the system functions for Box
  */
 export const boxMixin = compose(
   background,
-  bottom,
-  display,
-  height,
-  left,
-  minWidth,
-  maxWidth,
-  minHeight,
-  maxHeight,
+  border,
+  color,
+  flexbox,
```

Comment:
Does this change the default `display` of this component to be `flex`?

Reply:
Nah